doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -718,6 +718,16 @@ dd {
   padding-left: 1rem;
```

Comment:
Do we still need this then?

Reply:
Yea we need it for all the other `dd` elements. If we remove it we get this:
![Screen Shot 2020-06-19 at 5 59 24 PM](https://user-images.githubusercontent.com/5402633/85182452-b57cad80-b256-11ea-8c2b-4a7a5494ed54.png)